makefile

```diff
@@ -236,6 +238,41 @@ vendor-clean:
 	rm -rf vendor/github.com/docker/docker/vendor/golang.org/x/text/
 	rm -rf vendor/github.com/docker/docker/vendor/google.golang.org/genproto/
 	rm -rf vendor/github.com/docker/docker/vendor/google.golang.org/grpc/
+	rm -rf vendor/google.golang.org/genproto/googleapis/ads/
```

Comment:
OMG this is awful, sorry you had to do this.

Reply:
I have no idea why genproto decided to add _all_ their protobuf-generated code to this repo... oh well. I'm sure I could make it fancier by doing some `find` command here to remove all the excess packages, but I'm praying that this problem goes away with modules.